### PR TITLE
CB2STH: first basic integration test

### PR DESCRIPTION
**Pull request by anabelengp**
depends on https://github.com/telefonicaid/iot-qa-tools/pull/58

### CODE REVIEW

File: tests/steps/cb_steps.py [hunk 1]
```diff
@@ -78,8 +78,10 @@ def create_subscription(context):
                                                                             notify_conditions=notify_cond,
                                                                             throttling=throttling)
     # resp = cb.standard_subscribe_context_onchange(subscription_pl)
-    resp = eval("cb.standard_subscribe_context_" + context.table[0]['notify_type'].lower() + "(subscription_pl)")
+    resp = getattr(cb, "standard_subscribe_context_" + context.table[0]['notify_type'].lower())(subscription_pl)
```
Comment (XavierVal) [inline tests/steps/cb_steps.py]: +1 avoiding evals

**Comment by anabelengp** [comment]
* More cb2sth integration tests added.
* the updateContext step in cb_steps.py has been modified to allow creating several entities in the same updateContext request.
* the method remove_cb_entities has been removed because was confusing. It did not remove the entities, it only removed their attributes.
* cb2mysql.feature has been modified to
 * adapt it to the new updateContext method that allows creating several entities
 * add a delay before checking the tables in mysql
 * remove the tables after_feature.

File: tests/environment.py [hunk 2]
```diff
@@ -23,9 +23,10 @@
 import json
 import time
 from nose.tools import assert_true
+from pymongo import MongoClient
 from iotqatools.cb_utils import CbNgsi10Utils
 from common.test_utils import remove_mysql_databases
-from common.test_utils import bb_delete_method, devices_delete_method, remove_cb_entities
+from common.test_utils import bb_delete_method, devices_delete_method
```
Comment (XavierVal) [inline tests/environment.py]: this imports could be added to the previous one, right?

**Comment by XavierVal** [comment]
LGTM

**Pull request merged**